src/screens/ContactScreen/elements/Search.js

```diff
@@ -26,7 +26,6 @@ const Search = ({onPress, animatedValue, onChangeText, text, onClearText, isLoad
         returnKeyType="search"
         onSubmitEditing={onPress}
         textAlignVertical="center"
-        clearButtonMode="while-editing"
         blurOnSubmit={true}
       />
       <View style={styles.wrapperIcon}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `clearButtonMode="while-editing"` might affect the user experience as it provides a clear button while the user is editing the text. If this change was intentional and you have provided another way for users to clear the input, then it's fine. Otherwise, consider adding it back.
```diff
+        clearButtonMode="while-editing"
         returnKeyType="search"
         onSubmitEditing={onPress}
         textAlignVertical="center"
         blurOnSubmit={true}
       />
       <View style={styles.wrapperIcon}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->